cliquet/authorization.py

```diff
@@ -49,6 +61,15 @@ class RouteFactory(object):
     }
 
     def __init__(self, request):
+        # Prefix the user id with the authn policy type name.
+        user_id = request.authenticated_userid
+        # This comes from ``cliquet.initialization.setup_authentication()``.
+        authn_type = getattr(request, 'authn_type', None)
+        self.prefixed_userid = None
+        if user_id and authn_type:
+            self.prefixed_userid = '%s:%s' % (authn_type.lower(), user_id)
```

Comment:
OH ! This won't work with previous release of cliquet-fxa.
I should do test if already prefixed ! ``if not user_id.startswith(authn_type + '_')``